tests/drawer/test_draw_catalyst.py

```diff
@@ -17,6 +17,7 @@
 
 import pennylane as qml
 
+pytest.skip(allow_module_level=True)
```

Comment:
Should I unskip this now?

Reply:
Yep, that can be removed. With the catalyst addition, all should go green

Reply:
```suggestion

```